components/tooltip/index.css

```diff
@@ -25,7 +25,9 @@ governing permissions and limitations under the License.
 	--spectrum-tooltip-border-radius: var(--spectrum-corner-radius-400);
 
 	--spectrum-tooltip-font-size: var(--spectrum-font-size-75);
-	--spectrum-tooltip-line-height: var(--spectrum-line-height-100);
+
+	/* @todo set line-height using font size specific line-height tokens when they are finalized along with the new variable font. */
+	--spectrum-tooltip-line-height: 1.2;
```

Comment:
This is [similar to button](https://github.com/adobe/spectrum-css/blob/spectrum-two/components/button/index.css#L29) where the line-height token causes the overall height of the tooltip to be larger than the height token, so we need to adjust the line-height manually to make it work.